engines/bops_admin/config/routes.rb

```diff
@@ -6,7 +6,14 @@
   resource :dashboard, only: %i[show]
   resource :profile, only: %i[show edit update]
   resource :accessibility, only: %i[edit update], controller: "accessibility"
-  resource :notify, only: %i[edit update], controller: "notify"
+
+  resource :notify, only: %i[show edit update], controller: "notify" do
+    get :send_test, action: :send_test_new
+    post :send_test, action: :send_test_create
+
+    get :preview_letter, action: :preview_letter
+    post :preview_letter, action: :generate_preview
+  end
```

Comment:
Ideally I'd have preferred more restful routes like this:

``` ruby
resource :notify, only: %i[edit update], controller: "notify" do
  resource :email, only: %i[new create]
  resource :sms, only: %i[new create]
  resource :letter, only: %i[new create]
end
```

This allows you to create individual controllers for each type of notification and makes it easier for others to read the code in future.